tasks/check/check.mjs

```diff
@@ -0,0 +1,39 @@
+/* eslint-env node, es2021 */
+import core from '@actions/core'
+import { exec } from '@actions/exec'
+
+import { ok, fix } from '../release/prompts.mjs'
+
+console.log('-'.repeat(80))
```

Comment:
You might already know this --> I'm not sure if console.log will display within an GH Workflow run; I believe you need to use something like `core.info()`. 

Again, maybe this is intentional given dual-purpose. But things like...
`console.log(fix`You can fix this by running ${'yarn constraints --fix'}`)`
...seem helpful during CI run as well if indeed console.log isn't being outputted.

Reply:
I wasn't sure at first either but it seems to be showing up:
<img width="1522" alt="image" src="https://user-images.githubusercontent.com/32992335/151888764-f6c0b149-b708-4dc8-b7d5-c27baf750cf7.png">

Reply:
The tape goes over both and while the output is formatted slightly different, you get all the information

Reply:
🚀

Strange that I couldn't get it working for the Action I was attempting to build. Assuming user error...